README.md

```diff
@@ -30,6 +30,8 @@ Discussions that are short and stay on topic are much more likely to be read. If
 - Stick to the topic of the discussion. If a comment is tangential, or goes into detail on a subtopic, start a new discussion and link back.
 - Is your comment useful for others to read, or can it be adequately expressed with an emoji reaction to an existing comment?
 
+Language modifications which prevent specific syntax from occurring can probably be achieved with [an analyzer](https://docs.microsoft.com/en-us/visualstudio/extensibility/getting-started-with-roslyn-analyzers). Such changes are likely to be rejected by the language design committee to prevent increased language complexity.
```

Comment:
```suggestion
Language proposals which prevent specific syntax from occurring can be achieved with [a Roslyn analyzer](https://docs.microsoft.com/en-us/visualstudio/extensibility/getting-started-with-roslyn-analyzers). Proposals that only attempt to make existing syntax optionally illegal will be rejected by the language design committee to prevent increased language complexity.
```